packages/generator/src/options/options.ts

```diff
@@ -200,16 +183,22 @@ export const generatorOptionsCli = {
     describe:
       'Configuration file to ensure consistent names between multiple generation runs with updated / changed metadata files. Will be generated if not existent. By default it will be saved to/read from the input directory as "service-mapping.json".',
     type: 'string',
-    coerce: coercePathArg,
-    normalize: true
+    coerce: (
+      arg: string | undefined,
+      options: GeneratorOptions & { config?: string }
+    ) =>
+      coerceRequiredPathArg(
+        arg ? arg : join(options.inputDir.toString(), 'service-mapping.json'),
```

Comment:
[q] I would have expected the `join(options.inputDir.toString(), 'service-mapping.json')` as default - or is thisnot possible?

Reply:
This is not possible, because the option relies on another option (`inputDir`). I agree that maybe `coerce` should not handle defaults. However, these two activities are quite similar. Maybe, now that we have the `default` option in our hands, we could actually do more, like using functions for the default.

This can get more complicated though, because from a function it will be difficult to get a description of the default, so we probably would need ad *another* additional property.

For simplicity, I would probably rather go with the slightly dirty coerce function. If you disagree, I would be happy to discuss this and work on this in a separate PR.